mongo/src/main/java/com/redhat/lightblue/mongo/crud/IterateAndUpdate.java

```diff
@@ -58,6 +59,7 @@
 public class IterateAndUpdate implements DocUpdater {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IterateAndUpdate.class);
+    private static final Logger METRICS = LoggerFactory.getLogger("com.redhat.lightblue.metrics");
```

Comment:
Changed logger name to "metrics.com.redhat..."

On Thu, Aug 18, 2016 at 12:26 PM, Marek <notifications@github.com> wrote:

> In mongo/src/main/java/com/redhat/lightblue/mongo/crud/
> IterateAndUpdate.java
> <https://github.com/lightblue-platform/lightblue-mongo/pull/301#discussion_r75362307>
> :
>
> > @@ -58,6 +59,7 @@
> >  public class IterateAndUpdate implements DocUpdater {
> >
> >      private static final Logger LOGGER = LoggerFactory.getLogger(IterateAndUpdate.class);
> > +    private static final Logger METRICS = LoggerFactory.getLogger("com.redhat.lightblue.metrics");
>
> The labels are very generic. Might be a problem when we start adding
> measurements to other classes. I think you can leave those labels as is if
> you rename the logger, e.g.:
>
> LoggerFactory.getLogger("com.redhat.lightblue.metrics.IterateAndUpdate");
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lightblue-platform/lightblue-mongo/pull/301/files/0a7b67e01adcdbe4311266659c03337bce427ba5#r75362307>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADgDDfFCdl3LEOGSIEJrCvJ98avQ2ajWks5qhKPzgaJpZM4JnwQj>
> .
>